src/midje/emission/plugins/junit.clj

```diff
@@ -64,9 +64,12 @@
       (str (fact/file fact) ":" (fact/line fact))))
 
 (defn process-fact [fact]
-  (let [elapsed (/ (time/in-millis (time/interval (-> fact :attrs :start-time)
-                                                  (-> fact :attrs :stop-time)))
+  (let [start-time (-> fact :attrs :start-time)
+        stop-time (-> fact :attrs :stop-time)
+        elapsed (if (time/before? start-time stop-time) 
+                  (/ (time/in-millis (time/interval start-time stop-time))
                    1000.0)
```

Comment:
unfortunately there is no auto-linting on the project so we need to do it manually
here the `1000.0` needs one more level of indentation to match up with the first arg of the `/`

Reply:
Right! On it.

Reply:
Done.